templates/Web/_composition/MoleculerJS/Page.Moleculer.List.AddRoutes.WithCosmos.Mongo/server/services/pages.service_postaction.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
         // Call remove method that was loaded with `db.mixin.js`
         return this._remove(ctx, { id: ctx.params._id });
       }
-    },
-  //}]}
-  },
```

Comment:
@andremaz, could you revert the linting in the templates (specially in the _postaction) files? Removing this comma will make postaction fail as it relies on this line as context. 
As the file is not complete code file prettier gets confused I think. We should run prettier on the generated code, then adapt the templates accordingly instead.